### ISSUE TITLE
Predicted frame error

### ISSUE THREAD

**Issue by Gamesherd224** [Cartucho/OpenLabeling#48]
Sometimes when switching between frames with predicted bounding boxes, I get this error
Traceback (most recent call last):
  File "main.py", line 1007, in <module>
    tmp_img = draw_bboxes_from_file(tmp_img, annotation_paths, width, height)
  File "main.py", line 344, in draw_bboxes_from_file
    tree = ET.parse(ann_path)
  File "<string>", line 62, in parse
  File "<string>", line 38, in parse
cElementTree.ParseError: no element found: line 1, column 0

Can someone help me with this.
By the way this is the first time I am here so forgive me if I did something wrong

**Comment by Gamesherd224** [comment]
I now deleted that frame and the error disappeared, however, I am curious about why the error appeared.

**Comment by Gamesherd224** [comment]
Should I close this since this is not a problem anymore?

**Comment by Cartucho** [comment]
Hello! I am very sorry for the late response.

So I believe you had two `\n` in that file.

Do you still have that file with you? The one that was giving this error?

If you could give that information I could make sure that this will not happen again.

**Comment by Gamesherd224** [comment]
I still have the file but I have deleted that frame since then and changed the labels since then. I don't remember if I changed anything else, unfortunately. The name of the file is 'WIN_20190225_07_22_46_Pro', it is 4.63 MB. There is another video file there named 'WIN_20190330_10_20_46_Pro', but this is not the video with the error. Unfortunately, I do not think that any other piece of information could help since I have changed this part of the directory multiple times.
I hope this can help you in fixing this error.
